src/main/java/com/qoomon/maven/extension/gitversioning/VersioningModelProcessor.java

```diff
@@ -280,6 +287,16 @@ private ProjectVersion deduceProjectVersion(GAV gav, File gitDir) throws IOExcep
         }
     }
 
+    private void addPatternGroups(String label, String pattern, Map<String, String> dataMap) {
```

Comment:
please refactor this method to this one https://repl.it/repls/WoodenQuickwittedAmericanwarmblood